getinsomnia/insomnia
# feat: enable Certificate in pre-request scripting [INS-3379]

Changes:
- [x] enabled Certificate in pre-request scripting


### How to test:

Please run this script and check its output.
```
const { Certificate } = require('insomnia-collection');

	const cert = new Certificate({
			name: 'Certificate for example.com',
			matches: ['https://example.com'],
			key: { src: '/User/path/to/certificate/key' },
			cert: { src: '/User/path/to/certificate' },
			passphrase: 'iampassphrase',
	});

console.log(cert.name);
console.log(cert.key);
console.log(cert.cert);
console.log(cert.passphrase);
```
